example_data/sample_bundle.json

```diff
@@ -357,47 +398,60 @@
               "coding": [
                 {
                   "system": "http://www.genenames.org/geneId",
-                  "code": "HGNC:6990"
+                  "code": "HGNC:11111"
                 }
               ]
             },
-            "valueString": "MECP2"
+            "valueString": "SMC1A"
           },
           {
             "code": {
               "coding": [
                 {
                   "system": "http://hl7.org/evidence",
                   "code": "evidence",
-                  "display": "Evidence for classification of variant"
+                  "display": "Evidence for classification of variant1"
                 }
               ]
             },
-            "valueString": "absent from the gnomAD population database (PM2_Moderate)."
+            "valueString": "\u00ef\u0082\u00b7 Absent from the gnomAD population database (PM2_Moderate).  \u00ef\u0082\u00b7 Predicted null variant where loss of function is known disease mechanism (PVS1_very strong)."
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "81297-4",
+                  "display": "CNV"
+                }
+              ]
+            },
```

Comment:
So we need to model a CNV, but this a single base deletion isn't it? If so, then it's not a CNV

Reply:
variant in this observation is a SNV, not a CNV so will need to remove. @EthelEz to ask Natassa for example reports with CNVs in to then model the data. I'm happy to help also help with the modelling while we're working on the project

Reply:
suggest using this as the way to define the variant type https://loinc.org/LL4165-8